readme.md

```diff
@@ -71,7 +71,11 @@ There are two parts to creating an Application Link. The first is done in Jenkin
 To register a consumer: 
 1. In Jenkins go to **Jenkins** > **Manage Jenkins** > **Manage Bitbucket Server consumers**. 
 2. Select **Register new consumer**. 
-3. Enter consumer details. 
+3. Complete the form: 
+   - Consumer key: Enter a unique key that Jenkins can use to identify this Bitbucket instance. For example, bitbucket-syd.
```

Comment:
Hi @mhenschke-atl one thing we could do is to switch the order so it goes

- Consumer name
- Consumer key 
- Consumer secret 

I think this flow makes more sense as the user would enter the key and then the corresponding secret. 

What do you think?

Reply:
Sorry I missed this yesterday. Yes I agree, and I will switch the order of the fields on this branch as well.